src/k8s/pkg/component/component_test.go

```diff
@@ -200,3 +216,68 @@ func TestListComponentsWithReleases(t *testing.T) {
 		{Name: "two", Status: true},
 	}))
 }
+
+func TestComponentsInitialState(t *testing.T) {
+	g := NewWithT(t)
+
+	mockHelmClient, _, _ := mustCreateNewHelmClient(t, map[string]types.Component{
+		"one": {ReleaseName: "whiskas-1", Namespace: "default", ManifestPath: "chunky-tuna-0.1.0.tgz"},
+		"two": {ReleaseName: "whiskas-2", Namespace: "default", ManifestPath: "slim-tuna-0.1.0.tgz"},
+	})
+
+	components, err := mockHelmClient.List()
+	g.Expect(err).ShouldNot(HaveOccurred())
+	for _, component := range components {
+		g.Expect(component.Status).To(BeFalse(), "Expected all components to be initially disabled")
+	}
+}
+
+func TestEnableSingleComponent(t *testing.T) {
```

Comment:
What does `TestEnableSingleComponent` test that doesn't get covered in `TestEnableMultipleComponents`?

Reply:
Good point